cli/cmd/upgrade.go

```diff
@@ -50,9 +49,16 @@ var upgradeParams installUpgradeParams
 var keptnUpgradeChart *chart.Chart
 
 // installCmd represents the version command
-var upgraderCmd = NewUpgraderCommand(version.NewKeptnVersionChecker())
+var upgraderCmd = NewUpgraderCommand(version.NewKeptnVersionChecker(), helm.NewHelper(), kube.NewKubernetesUtilsKeptnNamespaceHandler(), common.NewUserInput(), credentialmanager.NewCredentialManager(assumeYes))
```

Comment:
I would suggest name it `NewUpgradeCommand` to be aligned with the `install.go`

Reply:
Done, thanks!